test/e2e_node/topology_manager_test.go

```diff
@@ -936,6 +940,90 @@ func runPreferClosestNUMATestSuite(ctx context.Context, f *framework.Framework,
 		podMap[podName] = pod
 	}
 
+	valiidatePreferClosestNUMAOptimalAllocation(ctx, f, podMap, numaNodes, distances)
+
+	deletePodsAsync(ctx, f, podMap)
+
+}
+
+func runPreferClosestNUMASubOptimalAllocationTest(ctx context.Context, f *framework.Framework, numaNodes int, distances map[int][]int) {
+	ginkgo.By("Admit two guaranteed pods. Both consist of 1 containers, each pod asks for cpus from 2 NUMA nodes. CPUs should be assigned from closest NUMA")
+	cntName := "ps-container-0"
+
+	// expect same amount of cpus per NUMA
+	cpusPerNUMA, err := getCPUsPerNUMANode(0)
+	framework.ExpectNoError(err)
+	if len(cpusPerNUMA) < 5 {
+		e2eskipper.Skipf("Less than 5 cpus per NUMA node on this system. Skipping test.")
+	}
+	podMap := make(map[string]*v1.Pod)
+	for podID := 0; podID < 2; podID++ {
+		// asks for all but one cpus from one less than half NUMA nodes, and half from the other
+		// plus add one less than half NUMA nodes, to accommodate for reserved cpus
+		numCores := ((numaNodes/2)-1)*(len(cpusPerNUMA)-1) + (len(cpusPerNUMA) / 2) + (numaNodes/2 - 1)
+		framework.ExpectNoError(err)
+
+		coresReq := fmt.Sprintf("%dm", numCores*1000)
+		ctnAttrs := []tmCtnAttribute{
+			{
+				ctnName:    "ps-container-0",
+				cpuRequest: coresReq,
+				cpuLimit:   coresReq,
+			},
+		}
+		podName := fmt.Sprintf("gu-pod-%d", podID)
+		framework.Logf("creating pod %s attrs %v", podName, nil)
```

Comment:
if the `attrs` are hardcoded `nil` let's remove them from the log

Reply:
fixed